public_html/css/index.css

```diff
@@ -154,3 +154,8 @@ h1 {
     /*border: 1px solid green;*/
     display: inline-block;
 }
+
+.status-div--reviewer {
+    margin-top: 10px;
+    white-space: nowrap;
+}
```

Comment:
I think we should have this in a smaller font. Same for timestamp. I have diff_timestamp and edit count wrapped in <small> but doing this in CSS is more elegant, definitely.